tasks/pipeline.py

```diff
@@ -134,17 +135,20 @@ def workflow_rules(gitlab_file=".gitlab-ci.yml"):
 
 
 @task
-def trigger(
-    _, git_ref=DEFAULT_BRANCH, release_version_6="nightly", release_version_7="nightly-a7", repo_branch="nightly"
-):
+def trigger(_, git_ref=DEFAULT_BRANCH, release_version_6="dev", release_version_7="dev-a7", repo_branch="dev"):
     """
     OBSOLETE: Trigger a deploy pipeline on the given git ref. Use pipeline.run with the --deploy option instead.
     """
 
     use_release_entries = ""
     major_versions = []
 
-    if release_version_6 != "nightly" and release_version_7 != "nightly-a7":
+    if (
+        release_version_6 != "nightly"
+        and release_version_7 != "nightly-a7"
+        and release_version_6 != "dev"
+        and release_version_7 != "dev-a7"
+    ):
```

Comment:
Is it really worth modifying this function?

Reply:
It's obsolete but just in case it's used somewhere, not needed but isn't a big deal to add.

Reply:
Not a big deal for sure, but if it's used it isn't doing anything :sweat_smile: